lib/src/configcat_client.dart

```diff
@@ -210,22 +210,22 @@ class ConfigCatClient {
     try {
       final result = await _getSettings();
       if (result.isEmpty) {
-        _errorReporter.error(
-            'Config JSON is not present. Returning defaultVariationId: \'$defaultVariationId\'.');
+        _errorReporter.error(1000,
+            'Config JSON is not present. Returning the `defaultVariationId` parameter that you specified in your application: \'$defaultVariationId\'.');
         return defaultVariationId;
       }
       final setting = result.settings[key];
       if (setting == null) {
-        _errorReporter.error(
-            'Variation ID not found for key $key. Here are the available keys: ${result.settings.keys.join(', ')}');
+        _errorReporter.error(1001,
+            'Failed to evaluate setting \'$key\' (the key was not found in config JSON). Returning the `defaultVariationId` parameter that you specified in your application: \'$defaultVariationId\'. Available keys: ${result.settings.keys.join(', ')}.');
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`